src/ClientWrapper.php

```diff
@@ -30,25 +28,32 @@ public function __construct(ClientOptions $clientOptions)
 
     public function getBasicClient(): Client
     {
-        if (empty($this->client)) {
-            $this->client = new Client($this->clientOptions->getClientConstructOptions());
-            $this->client->setRunId($this->clientOptions->getRunId());
-            $this->client->setBackendConfiguration($this->clientOptions->getBackendConfiguration());
+        if (empty($this->basicClient)) {
+            $this->basicClient = new Client($this->clientOptions->getClientConstructOptions());
+            $this->basicClient->setRunId($this->clientOptions->getRunId());
+            $this->basicClient->setBackendConfiguration($this->clientOptions->getBackendConfiguration());
         }
-        return $this->client;
+        return $this->basicClient;
     }
 
-    public function getBranchClient(): BranchAwareClient
+    public function getClientForBranch(string $branchId): BranchAwareClient
```

Comment:
Tady bych buď dodělal podporu aby to šlo volat s "default" branch a nebo to rovnou zakázal. Jinak se to o kus níž přetypuje na 0 a budem tam kde jsme byli.

Reply:
Hmm, a co kdyby to radši vyhodilo výjímku a trvalo to na numeric value? (i když typu string)

Reply:
No to jsem myslel tím

> a nebo to rovnou zakázal

Id default branche se dá získat pomocí toho wrapperu, takže klidně tak.

Reply:
ok, makes sense